### ISSUE TITLE
SPC geonode won't display uploaded layers/ documents or maps in catalog view

### ISSUE THREAD

**Issue by ksonda** [GeoNode/geonode#5926]
## Expected Behavior
After upload a layer or document, can view in "Explore Documents"/ "Explore Layers"

## Actual Behavior
Layers/ documents show up in layer of contents, but links in sidebar do not work, and the tiles showing available layers/ documents do not show up in the Explore Documents/ Explore Layers pages.

Note: When the images are pulled directly from dockerhub, there is no problem. The problem occurs when building the images

## Steps to Reproduce the Problem

  1. git clone https://github.com/geonode/geonode
  2. cd geonode/scripts/spcgeonode
  3. nano .env [set hostname, admin user/password]
  4. docker-compose -f docker-compose.yml up -d --build
  5. log in as admin
  6. Upload a layer or document
  7. Go to "Explore Layers" or "Explore Documents"

## Specifications

  - GeoNode version: 3.0
  - Installation method: SPC Geonode Docker
  - Platform: Ubuntu 18.04 LTS
  - Additional details:

**Comment by t-book** [comment]
@ksonda I cannot reproduce this behavior. Is this still happening for you with a fresh build? If not would you kindly close here? (@gannebamm the thumbs do not work in my test with documents is this a known issue with spc?)

![1585987200213](https://user-images.githubusercontent.com/20478652/78421852-55189f80-765b-11ea-9a2a-5ed191aa520d.png)
)

![1585987120484](https://user-images.githubusercontent.com/20478652/78421848-4f22be80-765b-11ea-8ff9-5a709b9d44d6.png)

**Comment by edsonflavio** [comment]
Helo, I have the same problem using this instalation mode. Docker, Linux Ubuntu and SPCGeonode.

**Comment by afabiani** [comment]
Wille take a look asap

**Comment by afabiani** [comment]
I cannot actually reproduce the error described in the description. You probably have outdated images.

The only issue I see until now is the one from @t-book about the auto-generated documents thumbnails. It looks like the SITEURL variable is not set correctly. This is something to be further investigated.

Moreover, with the latest geoserver wars updates, the edit feature functionality also has been fixed.

I took the occasion to push again the images on the docker repo which are now updated to 04/14/2020
